python/ola/OlaClient.py

```diff
@@ -1357,7 +1357,11 @@ def _RDMMessage(self, universe, uid, sub_device, param_id, callback, data,
     request.uid.device_id = uid.device_id
     request.sub_device = sub_device
     request.param_id = param_id
-    request.data = data
+    if sys.version >= '3.2':
+      request.data = eval(data)[0] if data else bytes(data, 'utf-8')
+    else:
+      # eval(data)[0] if data else data - broke (MultiDim), 2019-10-31
+      request.data = data  # works, 2019-10-31
```

Comment:
I think we can remove these comments can't we? The old code only had `request.data = data`.

I'm guessing this isn't necessary/required as one was an internal structure and one is a Protobuf call:
https://github.com/OpenLightingProject/ola/commit/255df6a6d86b030545384d79dce62d2509b8215a

Reply:
> I think we can remove these comments can't we?

Of course! (and sorry about that, did not carefully review that patch before posting; those comments were only intended to be internal)

Reply:
Done.